resources/publications.md

```diff
@@ -95,12 +93,35 @@ td:nth-child(2) {
 
 ## Metrics
 
+### Learnable metrics
+
+Directly optimize correlation with human judgments.
```

Comment:
*optimise* - European spelling

Reply:
One goal of Machine Translate is that all content will be... machine-translated.

So I worry a little bit about the way we describe each task.  We are using imperatives, which will cause trouble.

Can we think a way to avoid this?

e.g.

> For directly optimising ...

or

> To directly optimise ...

(And maybe no full-stop?)

Or make it a full sentence:

> Learnable metrics should **correlate with human judgements**.

Reply:
Thank you!
I think the last version can be confusing since here all the metrics should correlate with human judgements (see the picture in the comment below).

How about
_"Differently from the metrics that evaluate similarity between machine and human translations (see the following sections), learnable metrics optimise correlation with human judgements directly."_

Reply:
True

This kind of juxtaposition inside one sentence is a bit haram under our "Simple English" style guide.

I think we can break out the standalone clauses, parentheses etc.

> Learnable metrics optimise correlation with human judgements directly.
>
> They look at only the source sentence and translated sentence.  They do not [require a reference translation](#quality-estimation).